multitenant_provider/integration/pyproject.toml

```diff
@@ -6,6 +6,7 @@ authors = []
 
 [tool.poetry.dependencies]
 aries-cloudagent = { version = "0.10.3" }
```

Comment:
Think this was missed

Reply:
The `multitenant_provider` didn't like it when it was updated to a newer version of ACA-Py, the tests failed, so I reverted it back to the version that worked.
See:
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862684/job/19298755060
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862685/job/19298755070

Reply:
OK. I can have a look after this is merged.